alfresco-solr-query-analytics/README.md

```diff
@@ -0,0 +1,14 @@
+# important notes:
+
+The SolrJsonResultSet is extracted from:
+
+[sha256] 03bacac73bf2a5bad2600216b6249b673cea090c884cdfd18eb1ab2d51b97580  **alfresco-repository-21.20-sources.jar**
+
+The original checksum of the unaltered **SolrJSONResultSet.java** file:
+
+[sha256] b8ba53b23e58c973cf0e98f37396adeda9782dd3c359ca1aaf3fd2ddc546fc5c  **SolrJSONResultSet.java**
```

Comment:
We can discuss this, there might be an easier way, the text is quite verbose, I was just trying to make a point that might not be so helpful, not sure.